.github/workflows/container-image-test.yaml

```diff
@@ -0,0 +1,11 @@
+name: local-dev-test
+on: [pull_request, push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Very new to github actions, but seems like `unbuntu-latest` is the default for builds on github actions. Would there be value use a Red Hat based image ?

Reply:
These are the runners that GH natively supports:

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on

You do have the ability to provide your own runner if you would want too

Reply:
I assumed something like that. Just wanted to be sure.